codesign/bin/codesign.dart

```diff
@@ -19,7 +19,8 @@ const String kCodesignUserNameOption = 'codesign-username';
 const String kAppSpecificPasswordOption = 'app-specific-password';
 const String kCodesignAppStoreIdOption = 'codesign-appstore-id';
 const String kCodesignTeamIdOption = 'codesign-team-id';
-const String kGCloudDownloadUploadPathOption = 'google-cloud-download-upload-path';
+const String kGCloudDownloadPathOption = 'google-cloud-download-path';
```

Comment:
nit: You can make these shorter by saying `gcs-download-path`

Similarly, instead of saying `kGCloudDownloadPathOption`, it would be more specific to say `kGcsDownloadPathOption` as there's a few ways to download from GCP

Reply:
Sure, renamed.